### PR TITLE
feat: add `register` script to simplify registration testing

### PR DESCRIPTION
**Pull request by jawid-h**

### CODE REVIEW

File: test/functional/dpnsClient.spec.js [hunk 1]
```diff
@@ -48,23 +48,22 @@ async function registerIdentity(dpp, dashCoreApi, dapiClient, options = {}) {
   const publicKeyHash = PublicKey.fromBuffer(Buffer.from(pubKeyBase, 'base64'))
     ._getID();
 
-  await dashCoreApi.generate(500);
+  await dashCoreApi.generateToAddress(500, addressString);
   await dashCoreApi.sendToAddress(addressString, 10);
-  await dashCoreApi.generate(10);
```
Comment (thephez) [inline test/functional/dpnsClient.spec.js:53]: Was this supposed to be replaced with a `generateToAddress` like the other places `generate` was removed?

Comment (jawid-h) [reply test/functional/dpnsClient.spec.js]: @thephez We did, I don't know why you're still seeing it.

Comment (jawid-h) [reply test/functional/dpnsClient.spec.js:53]: @thephez Yeah, we've switched to `0.15` version of Dash Core in our tests.

Comment (thephez) [reply test/functional/dpnsClient.spec.js:53]: So this line should be added here, right?
`  await dashCoreApi.generateToAddress(10, addressString);`

**Review by shumkov — APPROVED** [review]
Good job!

**Pull request merged**